dev/functional/tests/cypress/page-objects/student-objects/WelcomeObject.ts

```diff
@@ -2,10 +2,7 @@ export default class WelcomeObject {
   checkCurrentUrl() {
     return cy
       .url()
-      .should(
-        "include",
-        "https://test-aest-sims.apps.silver.devops.gov.bc.ca/student"
-      );
+      .should("include", "https://dev-aest-sims.apps.silver.devops.gov.bc.ca/");
```

Comment:
This will be based on the URL we have passed. All testing is done based on DEV environment. Once Test Environment matches the DEV Env, we can run the full code on Test environment. We just need to update URL based on the environment ofcourse.